L1Trigger/L1TGlobal/plugins/TriggerMenuParser.cc

```diff
@@ -167,6 +167,13 @@ void l1t::TriggerMenuParser::setVecCorrelationWithOverlapRemovalTemplate(
   m_vecCorrelationWithOverlapRemovalTemplate = vecCorrelationWithOverlapRemovalTempl;
 }
 
+//set the AXO model version so it can be fetched from the GlobalProducer
+void l1t::TriggerMenuParser::setAXOL1TLModelVersion(const std::string& axol1tlmodelversion) {
+  if (m_axol1tlModelVersion.empty()) {  //only fills if has not yet been set
+    m_axol1tlModelVersion = axol1tlmodelversion;
+  }
```

Comment:
this is not really needed I suppose: this code is called once per condition, so in this example for 5 seeds it would be called 5 times. It doesn't really matter I guess if it is only filled once or set 5 times: the model version will be the same every time (there can only be one version per menu xml). I can change it to simply ` m_axol1tlModelVersion = axol1tlmodelversion;` if preferred

Reply:
per the suggestion, this has been changed to simply `m_axol1tlModelVersion = axol1tlmodelversion;`